build/build-viewer.js

```diff
@@ -6,33 +6,47 @@
 
 import {createRequire} from 'module';
 
-import {rollup} from 'rollup';
+import esbuild from 'esbuild';
+import * as terser from 'terser';
 
-import * as rollupPlugins from './rollup-plugins.js';
+import * as plugins from './esbuild-plugins.js';
 import {GhPagesApp} from './gh-pages-app.js';
 import {LH_ROOT} from '../root.js';
 
 const require = createRequire(import.meta.url);
 
 async function buildReportGenerator() {
-  const bundle = await rollup({
-    input: 'report/generator/report-generator.js',
+  const result = await esbuild.build({
+    entryPoints: ['report/generator/report-generator.js'],
+    write: false,
+    bundle: true,
+    minify: false,
     plugins: [
-      rollupPlugins.shim({
+      plugins.umd('ReportGenerator'),
+      plugins.replaceModules({
         [`${LH_ROOT}/report/generator/flow-report-assets.js`]: 'export const flowReportAssets = {}',
       }),
-      rollupPlugins.nodeResolve(),
-      rollupPlugins.removeModuleDirCalls(),
-      rollupPlugins.inlineFs({verbose: Boolean(process.env.DEBUG)}),
+      plugins.ignoreBuiltins(),
+      plugins.bulkLoader([
+        plugins.partialLoaders.inlineFs({verbose: Boolean(process.env.DEBUG)}),
+        plugins.partialLoaders.rmGetModuleDirectory,
+      ]),
     ],
   });
 
-  const result = await bundle.generate({
-    format: 'umd',
-    name: 'ReportGenerator',
-  });
-  await bundle.close();
-  return result.output[0].code;
+  // @ts-expect-error placed here by the umd plugin.
+  let code = result.outputFiles[0].textUmd;
+
+  if (!process.env.DEBUG) {
+    code = (await terser.minify(code, {
```

Comment:
Why use terser here instead of esbuild minify? Are there page functions in `report-generator.js`?